src/main/java/org/cryptomator/ui/fxapp/UpdateCheckerModule.java

```diff
@@ -63,6 +66,7 @@ static HttpRequest provideCheckForUpdatesRequest(Environment env) {
 		return HttpRequest.newBuilder() //
 				.uri(LATEST_VERSION_URI) //
 				.header("User-Agent", userAgent) //
+				.timeout(java.time.Duration.ofSeconds(3))
```

Comment:
Can't you import this class?

Reply:
Because we already import [javafx.util.Duration](https://openjfx.io/javadoc/21/javafx.base/javafx/util/Duration.html).